### PR TITLE
Pulsar adjustments

### PR DESCRIPTION
**Pull request by mauricioszabo**
Basically, re-adjusting things here that are still pointing to "atom home" to start searching on "pulsar home"

### CODE REVIEW

File: src/rebuild-module-cache.coffee [hunk 1]
```diff
@@ -20,7 +20,7 @@ class RebuildModuleCache extends Command
       Usage: apm rebuild-module-cache
 
       Rebuild the module cache for all the packages installed to
-      ~/.atom/packages
+      ~/.range/packages
```
Comment (confused-Techie) [inline src/rebuild-module-cache.coffee:23]:
```suggestion
      ~/.pulsar/packages
```

This seems like the value we would expect here?

**Review by confused-Techie — CHANGES_REQUESTED** [review]
Added a single comment I think got missed. Otherwise this looks great.

File: src/rebuild-module-cache.coffee (hunk 1 above)
Comment (mauricioszabo) [reply src/rebuild-module-cache.coffee]: Great catch

**Review by confused-Techie — APPROVED** [review]
Thanks for being so fast on my suggestion. This looks like a great contribution to me

**Comment by confused-Techie** [comment]
Also I believe the reason that test is failing, is because its expecting a different return, which you've changed to `The Pulsar or Atom engine range in the package.json file is invalid:` because that's what's accurate, but the test expects `The Atom engine range in the package.json file is invalid:`. Should be able to just update the string it expects in the test to this

**Comment by confused-Techie** [comment]
Yup, should just change the string expected in `spec/publish-spec.coffee` Line 77

**Pull request merged**